src/Interfaces/AbstractHyperrectangle.jl

```diff
@@ -570,3 +571,34 @@ The `Hyperrectangle` that corresponds to the rectification of `H`.
 function rectify(H::AbstractHyperrectangle)
     Hyperrectangle(low=rectify(low(H)), high=rectify(high(H)))
 end
+
+"""
+    volume(H::AbstractHyperrectangle{N}) where {N<:Real}
+
+Return the volume of a hyperrectangular set.
+
+### Input
+
+- `H` -- hyperrectangular set
+
+### Output
+
+The volume of ``H``.
+
+### Algorithm
+
+The volume of the ``n``-dimensional hyperrectangle ``H`` with vector radius ``r`` is
+``2ⁿ ∏ᵢ rᵢ`` where ``rᵢ`` denotes the ``i``-th component of ``r``.
+"""
+function volume(H::AbstractHyperrectangle{N}) where {N<:Real}
+    vol = mapreduce(x -> 2x, *, radius_hyperrectangle(H))
+    return vol
+end
+
+@inline function _vol_prod(H::AbstractHyperrectangle{N}, n) where {N<:Real}
+    vol = one(N)
+    for i in 1:n
+        vol *= 2 * radius_hyperrectangle(H, i)
+    end
+    return vol
+end
```

Comment:
This function is now unused, right?

Reply:
Yes. Do you want to remove it, or do you suggest to keep it?

Reply:
Yes. Do you want to remove it, or do you suggest to keep it?

Reply:
I see no point in keeping it if you have decided against using it.